src/setOfPeople.js

```diff
@@ -38,7 +38,17 @@
  * @return {Set}
  */
 function setOfPeople(people) {
-  // write code here
+  const uniquePeople = people.reduce((uniquePeople, person) => {
```

Comment:
https://github.com/mate-academy/js_n-for-names/pull/21#discussion_r275247892
`uniquePeople `